index.html

```diff
@@ -183,16 +193,17 @@
       </tbody>
     </table>
 
-    <p class="note">The <code>gAMA</code> and embedded ICC profile are provided solely for compatibility with processors that do
-    not conform to this specification.</p>
+    <p class="note">The <code>gAMA</code> chunk, <code>cHRM</code> chunk, and embedded ICC profile specified in the table above are
+    not necessarily accurate, and, even if accurate, their processing is not required to reproduce the image as intended by the
+    author. They are provided for graceful fallback for implementations that do not recognize the semantics associated with a
```

Comment:
> I meant in the case where a system with HDR capability uses the fallback, because it does not follow this spec..

I do not think it is possible to predict exactly what HDR systems will do when processing an ICC profile that was not designed for HDR systems... some might map the 80 nits of the ICC profile to what peak SDR luminance would be in SDR mode, others might map it to 80 nits and others might map it to peak HDR luminance.